### PR TITLE
[BI-2102] Accessibility: Make Color Contrast Meet AA Standard

### PR DESCRIPTION
**Pull request by davedrp**
# Description
[BI-2102](https://breedinginsight.atlassian.net/browse/BI-2102) Accessibility: Make Color Contrast Meet AA Standard

# Dependencies
bi-api: develop branch

# Testing
NOTE: To test this install the Siteimprove Accessibility Checker browser plugin found at https://chromewebstore.google.com/detail/siteimprove-accessibility/djcglbmbegflehmbfleechkjhmedcopn?pli=1

**Go to _BrAPI Information_ page.**
EXPECTED RESULT
The Siteimprove Accessibility Checker should not report the issue `Color contrast does not meet minimum requirement`

 **Go to the _Jobs_ page.**
 EXPECTED RESULT
 The Siteimprove Accessibility Checker should not report the issue `Color contrast does not meet minimum requirement`

 **Go to any page that includes a buefy table with pagination (for example, the _Gemplasm_ page).**
 EXPECTED RESULT
 The Siteimprove Accessibility Checker should not report the issue `Color contrast does not meet minimum requirement`

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have tested my code and ensured it meets the acceptance criteria of the story
- [ ] I have create/modified unit tests to cover this change
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to documentation
- [ ] I have run TAF: _\<link to TAF run>_

[BI-2102]: https://breedinginsight.atlassian.net/browse/BI-2102?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by dmeidlin — APPROVED** [review]

File: src/assets/scss/main.scss [hunk 1]
```diff
@@ -27,14 +27,18 @@
...
 $blue: #1B64DD;
 
 // Custom colors
 $purple: #763FCE;
 $magenta: #9c27b0;
 $applegreen: #C1ED2F;
+$lt-turquoise: #65c2c4;
 $turquoise: #3FB3B6;
+$dark-turquoise: #297577;
+//$dark-turquoise: #2d8082;
```
Comment (mlm483) [inline src/assets/scss/main.scss:40]: Please remove or document why this commented-out line is here.

File: src/assets/scss/main.scss.bak [hunk 2]
```diff
@@ -0,0 +1,1037 @@
+/*!
```
Comment (mlm483) [inline src/assets/scss/main.scss.bak:1]: I don't think this file should be in git. If we really need to find the old values, we can look at the commit history.

**Review by mlm483 — CHANGES_REQUESTED** [review]

File: src/assets/scss/main.scss (hunk 1 above)
Comment (davedrp) [reply src/assets/scss/main.scss]: done

File: src/assets/scss/main.scss.bak (hunk 2 above)
Comment (davedrp) [reply src/assets/scss/main.scss.bak]: done

**Review by mlm483 — APPROVED** [review]
🚀

**Review by dmeidlin — APPROVED** [review]

**Pull request merged**